<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>477</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>520</width>
    <height>477</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>520</width>
    <height>477</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
    <stylestrategy>PreferAntialias</stylestrategy>
   </font>
  </property>
  <property name="windowTitle">
   <string>GitHub Web-scrapper PyQt6</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>logo.png</normaloff>logo.png</iconset>
  </property>
  <widget class="QPushButton" name="pushButton_scrap">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Получить данные</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_exit">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>440</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Выйти</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_username">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>meirbek-dev</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_login">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>6</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Введите пользователя GitHub:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_info">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>5</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Данные аккаунта:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_info">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>30</y>
     <width>261</width>
     <height>401</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="accessibleName">
    <string/>
   </property>
   <property name="accessibleDescription">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextSelectableByMouse</set>
   </property>
   <property name="placeholderText">
    <string>Здесь будут отображаться данные о репозиториях</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_clear">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>440</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Очистить</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_status">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>231</width>
     <height>321</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
     <strikeout>false</strikeout>
     <stylestrategy>PreferAntialias</stylestrategy>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
